src/app/search-results/search-results.component.ts

```diff
@@ -3,23 +3,25 @@
 /* - subscribes to ActivatedRoute instance for live query / result data
 */
 import { HttpErrorResponse } from '@angular/common/http';
-import { Component, OnInit } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { DatasetSearchView } from '../_models';
 import { DatasetsService, DocumentTitleService } from '../_services';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'search-results',
   templateUrl: './search-results.component.html',
   styleUrls: ['./search-results.component.scss']
 })
-export class SearchResultsComponent implements OnInit {
+export class SearchResultsComponent implements OnDestroy, OnInit {
   searchString: string;
   currentPage = 0;
   isLoading = false;
   hasMore = false;
   query: string;
   results: DatasetSearchView[];
+  subParams: Subscription;
```

Comment:
Another candidate to be SubscriptionManagered?

Reply:
Done